tests/BootstrapperTest.php

```diff
@@ -121,7 +128,13 @@
 
     // Asset central is still the same
     expect(Cache::get('foo'))->toBe('central');
-});
+
+    // Reset the static property
+    CacheManager::$addTags = false;
+})->with([
+    'tagging' => CacheTagsBootstrapper::class,
+    'prefixing' => PrefixCacheTenancyBootstrapper::class,
```

Comment:
Are the keys used in any way? If not, we can simplify this into a normal array.

Reply:
Same in GlobalCacheTest

Reply:
They aren't, so yeah, we can just use a normal array